great_expectations/checkpoint/actions.py

```diff
@@ -606,6 +606,41 @@ def _validate_renderer(cls, renderer: dict | OpsgenieRenderer) -> OpsgenieRender
             renderer = _renderer
         return renderer
 
+    @override
+    def v1_run(self, checkpoint_result: CheckpointResult) -> dict:
+        validation_success = checkpoint_result.success
+        checkpoint_name = checkpoint_result.checkpoint_config.name
+
+        if (
+            self.notify_on == "all"
```

Comment:
I was thinking about this in the other PR but didn't mention it because it was just reusing existing code: maybe it's just me but I find these longer lines of `or` / `and` hard to parse in python. What about making intermediate variables for the `and` parts, and then just having this condition do the `or`s if that makes sense.